README.rst

```diff
@@ -1,1692 +1,132 @@
-Suppose that you start an instance with ``advertise_uri`` set to
-``localhost:3302``. Addressing that instance as ``<instance-host>:3302`` from a different
-instance won't work, because other instances will only recognize it as ``localhost:3302``.
-
-If you specify only a port, ``cartridge`` will use an auto-detected IP.
-In this case you have to configure Docker networks to set up inter-instance communication.
-
-You can use Docker volumes to store instance snapshots and xlogs on the
-host machine. If you updated your application code, you can create a new image for it,
-stop the old container, and start a new one using the new image.
-
-.. _cartridge-cli-special-files:
-
-Build and packaging files
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Put these files in your application directory to control the packaging process.
-See the examples below.
-
-* ``cartridge.pre-build`` is a script that runs before ``tarantoolctl rocks make``.
-  The main purpose of this script is to build non-standard rocks modules
-  (for example, from a submodule).
-  Must be executable.
-
-* ``cartridge.post-build`` is a script that runs after ``tarantoolctl rocks make``.
-  The main purpose of this script is to remove build artifacts from the final package.
-  Must be executable.
-
-.. _cartridge-cli-example-cartridge-prebuild:
-
-Example: cartridge.pre-build
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-..  code-block:: bash
-
-    #!/bin/sh
-
-    # The main purpose of this script is to build non-standard rocks modules.
-    # It will run before `tarantoolctl rocks make` during application build.
-
-    tarantoolctl rocks make --chdir ./third_party/my-custom-rock-module
-
-.. _cartridge-cli-example-cartridge-postbuild:
-
-Example: cartridge.post-build
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-..  code-block:: bash
+*   `Stopping the application locally <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#stopping-the-application-locally>`__
+*   `Checking instance status <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#checking-instance-status>`__
+*   `Displaying logs <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#displaying-logs>`__
+*   `Cleaning instance files <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#cleaning-instance-files>`__
+*   `Repairing a cluster <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#repairing-a-cluster>`__
+*   `Packaging your application <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#packaging-the-application>`__
 
-    #!/bin/sh
+    -   `Building a distribution <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#build-directory>`__
+    -   `Packing a TGZ <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#repair-commands>`__
+    -   `Packing an RPM or DEB distribution <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#rpm-and-deb>`__
```

Comment:
I like this structurization: packaging section and tgz, rpm, deb, docker subsection. However there are several comments around:

1. Packaging into a Docker image is missed in this list.
2. The ToC on the website does not follow this structure.
3. The link to the build step of the `pack` command looks useless in this list (why one would follow this link?). Maybe the sebsection is in the inappropriate place itself.